config/config.go

```diff
@@ -600,6 +614,11 @@ func (c *RemoteWriteConfig) UnmarshalYAML(unmarshal func(interface{}) error) err
 			return errors.New("empty or null relabeling rule in remote write config")
 		}
 	}
+	for header := range c.Headers {
+		if _, ok := unchangeableHeaders[strings.ToLower(header)]; ok {
+			return errors.Errorf("%s is an unchangeable header", header)
```

Comment:
Would you provide a custom message for `authorization` since you can change that header by using a different field. For example: `use basic_auth or bearer_token for the authorization header`.

Reply:
Thanks, fixed those!